components/runtime/spidermonkey/patches/05-pgo-ss12_2.patch

```diff
@@ -0,0 +1,18 @@
+diff --git a/js/src/configure b/js/src/configure
+index 0e98daf..11bb34f 100755
+--- a/configure
++++ b/configure
+@@ -14560,10 +14560,10 @@ if test -n "$INTEL_CC"; then
+ fi
+ 
+ if test "$SOLARIS_SUNPRO_CC"; then
+-  PROFILE_GEN_CFLAGS="-xprofile=collect:$_objdir/$enable_application"
```

Comment:
Relevant only for Studio

Reply:
It is protected so we can use the patch or remove it. It doesn't matter.